src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/CaseDataController.java

```diff
@@ -115,10 +118,8 @@
 
     private void setData(final String authToken, final Map<String, Object> caseData) {
         if (idamService.isUserRoleAdmin(authToken)) {
-            log.info("Admin users.");
```

Comment:
from my experience it's always better to have more logs.

What I'd suggest here is to add an id or some reference from ccd case data provided. Otherwise it's difficult to find out what request the log comes from.